src/translations/pages/contact.ts

```diff
@@ -1018,9 +1018,9 @@ export const Contact = {
       ),
       errorMessages: {
         empty: _(
-          'Vennligst fyll ut informasjon om deg',
-          'Please provide your information',
-          'Vennligst fyll ut din informasjon',
+          'Vennligst fyll ut etternavnet ditt',
+          'Please enter your lastname',
```

Comment:
```suggestion
          'Please enter your last name',
```

"Surname" eller "family name" er og alternativer. Hva er brukt tidligere, i app/nettbutikk feks?